.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
just for reference, including the github actions [hosted runner virutal-environments "installed softwares" list for ubuntu here, including `go` itself](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#go). just sharing the list for link reference.

this looks good to me 👍  future would presumably just be to plop in `macos-latest`, `windows-latest` in here as well. good stuff.